tests/integration/cattletest/core/test_svc_discovery.py

```diff
@@ -295,21 +311,13 @@ def test_remove_inactive_service(super_client, admin_client, sim_context, nsp):
     assert instances[0].state == "running"
 
     # deactivate service
-    service = wait_success(admin_client, service.deactivate(), 120)
+    service = wait_success(admin_client, service.deactivate())
     assert service.state == "inactive"
 
     # remove service
-    service = wait_success(admin_client, service.remove(), 120)
+    service = wait_success(admin_client, service.remove())
     assert service.state == "removed"
-    instances = super_client. \
-        list_container(name="compose_" + service.name + "_" + "1")
-    assert len(instances) == 1
-    assert instances[0].state == "removed"
-    instance_service_map = super_client. \
```

Comment:
actually, i don't think it's even necessary to add it back in.  but was
wondering whether there was any particular reason in the first place

On Wed, Apr 8, 2015 at 2:10 PM, Alena Prokharchyk <notifications@github.com>
wrote:

> In tests/integration/cattletest/core/test_svc_discovery.py
> <https://github.com/rancherio/cattle/pull/337#discussion_r28013651>:
>
> >      assert service.state == "removed"
> > -    instances = super_client. \
> > -        list_container(name="compose_" + service.name + "_" + "1")
> > -    assert len(instances) == 1
> > -    assert instances[0].state == "removed"
> > -    instance_service_map = super_client. \
>
> I can re-add this check later. I removed it because making sure the
> instance is removed, is sufficient enough; and with the async changes I had
> to add more logic for making sure that the mapping is gone.
> wait_for_condition can't handle map objects because they don't support
> "reload" method
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rancherio/cattle/pull/337/files#r28013651>.
>